### PR TITLE
Clean up closed.tsx to carefully reference correct data related to user winnings

### PR DESCRIPTION
**Pull request by KadenZipfel**
Closes: https://github.com/protofire/omen-exchange/issues/1776

To test: create markets and carefully track your shares to make sure the correct data is returned. 
Kinda screwed up the commit history :/

### CODE REVIEW

File: app/src/services/cpk.ts [hunk 1]
```diff
@@ -1360,11 +1360,13 @@ class CPKService {
         // get mainnet relay signer
         const to = await this.cpk.ethLibAdapter.signer.signer.getAddress()
 
+        const withdrawAmount = amount.sub(RELAY_FEE)
```
Comment (hexyls) [inline app/src/services/cpk.ts:1363]: Just gotta remove the relay fee here, must have accidentally stuck around in the merge.

**Review by hexyls — APPROVED** [review]
Wasn't total sure how to test the changes but it does appear to work and I could redeem the correct amount:

Screen Shot 2021-03-29 at 3 36 56 pm

**Review by pimato — CHANGES_REQUESTED** [review]
The app crashes on this market: http://localhost:3000/#/0x5e3aba6d741ed0d289b461508472ceacd64be339
Bildschirmfoto 2021-03-29 um 09 13 14

on master everything seems to load just fine:
Bildschirmfoto 2021-03-29 um 09 15 05

**Review by Mi-Lan — APPROVED** [review]
I've tested bunch of cases and could't find that error again. Forgot to note which market I was getting it on. Heres couple of cases where it works.

- Also tip for testing you can use history table to check if shares and amounts won align! Found that its pretty accurate! GJ

Screenshot 2021-03-30 at 17 29 44
Screenshot 2021-03-30 at 17 02 31

**Review by pimato — APPROVED** [review]

**Pull request merged**